src/figuresClasses.ts

```diff
@@ -1,11 +1,90 @@
-export interface Figure {}
+// export interface Figure {}
 
-export class Triangle implements Figure {}
+// export class Triangle implements Figure {}
 
-export class Circle implements Figure {}
+// export class Circle implements Figure {}
 
-export class Rectangle implements Figure {}
+// export class Rectangle implements Figure {}
 
-export function getInfo(figure): string {
-  return typeof figure;
+// export function getInfo(figure): string {
+//   return typeof figure;
+// }
+
+enum Shape {
+  Triangle = 'triangle',
+  Circle = 'circle',
+  Rectangle = 'rectangle',
```

Comment:
Enum constant names should start with a capital letter. Please adjust the enum values to start with a capital letter for consistency and to follow best practices.